### ISSUE TITLE
Question: When can we expect 0.2.0 release?

### ISSUE THREAD

**Issue by ktoso** [scala-ide/scalariform#223]
Hi there,
we were thinking to make the jump to the latest version with Akka but it [seems](https://github.com/akka/akka/pull/19936) we'd want to use some of post 1.8 features to not change too much of the codebase.

Is there a more specific plan in terms of timing of the 0.2.0?

**Comment by sschaef** [comment]
No, we don't have a release date for the next release. It will happen when the tickets in this milestone are finished: https://github.com/scala-ide/scalariform/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0 I wouldn't expect a release too soon.

**Comment by ktoso** [comment]
I see, so the 0.2.0 is not actually planned, and we're waiting on the 1.0 to happen?

**Comment by sschaef** [comment]
yes

**Comment by farmdawgnation** [comment]
I want to chime in and say that a 0.2.0 release of some sort with an update to the sbt plugin would be really useful to my team. :/

**Comment by sschaef** [comment]
unlikely that it will happen at all in the current situation. I'm not motivated in doing all of the remaining work anymore. Therefore, it depends if someone shows up as a maintainer for the project.

**Comment by jkinkead** [comment]
I'm happy to be a primary maintainer on this. Most of the 0.2.0 bugs are fixed in pending PRs from me; I can move forward with merging them & finishing up the remaining work if you don't mind me taking charge.

**Comment by sschaef** [comment]
Sure, go ahead. You already have write permissions. Feel free to move the project forward in the way how you like it. @fommil probably is of the same opinion.

**Comment by fommil** [comment]
Yeah go for it. I'd really hoped we'd find the time to create a good regression test harness against big projects but it became very low priority for me.

**Comment by ckozak** [comment]
Was there any progress on releasing 0.2?

**Comment by sschaef** [comment]
No, there wasn't. The project is basically unmaintained at the moment.

**Comment by fernandoacorreia** [comment]
@jkinkead From the conversation above, it seems that the project is open for a new maintainer. Are you still willing to do it? There's a few of people using it at the company I work for and I think we could contribute with testing and perhaps some coding too.

**Comment by jkinkead** [comment]
Sorry, my summer has been busy. I'll have more time this fall for maintenance.

Right now there's a `1.0` milestone that was created by the Scala IDE team that has a few blockers on it: https://github.com/scala-ide/scalariform/issues?q=is%3Aopen+is%3Aissue+milestone%3A1.0

The biggest is #153 .

I'd be happy to release a `0.2` release as a patch-fix, but I don't believe I have permissions to publish. Is this something you could set up, @sschaef ?

**Comment by sschaef** [comment]
Do you have a sonatype account?

**Comment by jkinkead** [comment]
Yep, it's `jkinkead`. :)

**Comment by jkinkead** [comment]
Are you able to set up these permissions, @sschaef , for the sonatype account `jkinkead`?

**Comment by sschaef** [comment]
Sorry for the delay, I just created a request to grant permissions: https://issues.sonatype.org/browse/OSSRH-25005

**Comment by jkinkead** [comment]
Awesome, thank you!

Are you and / or @fommil admins on [sbt-scalariform](https://github.com/sbt/sbt-scalariform) as well? I'd want to release that at the same time, but I don't believe I have write permissions on it.

It would also let me get the suite of integration tests in that's blocking a `1.0` release.

**Comment by sschaef** [comment]
I just sent an invite

**Comment by jkinkead** [comment]
Great, thanks again!

**Comment by ryan-williams** [comment]
Any word on this? Some breadcrumbs led me here: https://github.com/sbt/sbt-scalariform/issues/45 https://github.com/sbt/sbt-scalariform/issues/36

**Comment by mirkof** [comment]
Is there any update on this? Looks like milestone blockers for 1.0 haven't been touched at all...
Can we at least get 0.2?

**Comment by godenji** [comment]
> When can we expect 0.2.0 release?

Now

> "org.scalariform" %% "scalariform" % "0.2.0"

**Comment by farmdawgnation** [comment]
![mic drop](https://media.giphy.com/media/3o7qDEq2bMbcbPRQ2c/giphy.gif)
